includes/admin/meta-boxes/views/html-unknown-related-orders-row.php

```diff
@@ -18,10 +18,17 @@
 		echo sprintf( esc_html_x( '#%s', 'hash before order number', 'woocommerce-subscriptions' ), esc_html( $order_id ) );
 		?>
 		<div class="wcs-unknown-order-info-wrapper">
-			<a href="https://docs.woocommerce.com/document/subscriptions/orders/#section-8">
+			<a href="https://woocommerce.com/document/subscriptions/orders/#why-are-some-orders-in-the-related-orders-table-not-linking-to-the-order">
 				<?php
-				// Translators: The %1 placeholder is the translated order relationship ("Parent Order"), %2 placeholder is a <br> HTML tag.
-				echo wcs_help_tip( sprintf( __( 'This %1$s couldn\'t be loaded from the database. %1$s Click to learn more.', 'woocommerce-subscriptions' ), $relationship, '</br>' ) ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+				echo esc_html(
```

Comment:
How about removing this esc_html() call? 

It's escaping the HTML that's intended to be printed. The function [wcs_help_tip()](https://github.com/Automattic/woocommerce-subscriptions-core/blob/trunk/includes/wcs-compatibility-functions.php#L18-L37) already escapes it via [wc_help_tip()](https://hooks.wbcomdesigns.com/reference/functions/wc_help_tip/). We _may_ want to escape the `$relationship` variable alone.

| With escaping | Without escaping |
|--------|--------|
| <img width="400" alt="image" src="https://github.com/user-attachments/assets/ec6baab4-3900-41c4-b7df-04d87b1e8652"> | <img width="400" alt="image" src="https://github.com/user-attachments/assets/9cf47858-2357-44c6-a12e-cca532b2c58b"> | 

We could then add the comment `// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped`  above the `echo` statement to silence phpcs' warning.

Reply:
Thanks Danae, I'd had a feeling I was getting something wrong on that one. I appreciate you setting me right. It looks like the comment was already in the doc, but me breaking the function to multiple lines caused it to not get recognized at the right time by phpcs. Should be fixed now. 🤞